Sources/SwiftToolchain/ToolchainManagement.swift

```diff
@@ -236,7 +236,7 @@ public class ToolchainSystem {
   }
 
   public struct SwiftPath {
-    public var verison: String
```

Comment:
public var of public struct, so this might be breaking for someone?

Reply:
it's fine :)